---
geometry_msgs/PointStamped pose
